tasks/main.yml

```diff
@@ -121,7 +121,7 @@
   - name: Enable Receptor systemd units
     systemd:
       unit: "receptor@{{ item }}.service"
-      enabled: yes
+      enabled: "{{ cloud_connector_enabled }}"
```

Comment:
Wouldn't this just prevent it from being started after a machine boot? From what I remember about ansible's `systemd` module it would still keep the service running or start it if it is not running

Reply:
Great point, I updated the state line to account for this.